src/matrix/room/timeline/Timeline.js

```diff
@@ -199,8 +200,10 @@ export class Timeline {
         if (!this._localEntries?.hasSubscriptions) {
             return;
         }
-        // find any local relations to this new remote event
-        for (const pee of this._localEntries) {
+        // find any local relations to these new remote events or maybe these
+        // new remote events reference one of the other new remote events we have.
+        const entryList = new ConcatList(entries, this._localEntries);
+        for (const pee of entryList) {
```

Comment:
This makes emoji reactions show up on messages.

(the `m.reaction` events are in the list of events we're rendering)